### PR TITLE
feat: Configure Kuiper for secure message bus

### PR DESCRIPTION
**Pull request by lenny-intel**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/master/.github/Contributing.md.

## What is the current behavior?
Kuiper doesn't have secure message bus credentials for Redis MessageBus

## What is the new behavior?
When in secure mode and message bus is `redis`, write Kuiper config file with Redis credentials.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## New Imports

- [ ] Yes
- [x] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

**Review by jim-wang-intel — CHANGES_REQUESTED** [review]
would suggest using golnag's text/template for this perfect case

File: internal/security/secretstore/secure-messagebus.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	}
+
+	return nil
+}
+
+func configureKuiperForSecureMessageBus(credentials UserPasswordPair, configPath string, lc logger.LoggingClient) error {
+	kuiperEdgexConfig := kuiperConfigTemplate
+	kuiperEdgexConfig = strings.Replace(kuiperEdgexConfig, usernamePlaceholder, credentials.User, -1)
+	kuiperEdgexConfig = strings.Replace(kuiperEdgexConfig, passwordPlaceholder, credentials.Password, -1)
+
```
[the remark this answers is not in the document]
Comment (lenny-intel) [reply internal/security/secretstore/secure-messagebus.go]: Done

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Comment by lenny-intel** [comment]
recheck

**Pull request merged**